.circleci/config.yml

```diff
@@ -130,7 +130,7 @@ jobs:
           no_output_timeout: 15m
           command: |
             set -em
-            VERSION=64 BROWSER=firefox npm run test:integration
+            VERSION=65 BROWSER=firefox npm run test:integration
```

Comment:
Can we remove the version indicator and have it use the latest? The version 64 was added due to 65 breaking.

Reply:
Was gonna ask if this should be latest, removed